Gemfile

```diff
@@ -0,0 +1,51 @@
+source 'https://rubygems.org'
+git_source(:github) { |repo| "https://github.com/#{repo}.git" }
+
+ruby '3.2.1'
```

Comment:
@zakiyousufi, This happens because we may all have different versions of ruby installed on our local machine. To bypass this, when testing other people's code, simply update the ruby version in the gemfile to the ruby version on your machine